modules/deb.sh

```diff
@@ -221,6 +221,22 @@ function deb_get_and_install_package() {
     deb_install_package "work/deb-packages/${DEB_NAME}" "work/rootfs/"
 }
 
+#  Install binary files of a deb package into work/deb-packages from a local directory
+#  $1 - Directory path
+#
+function deb_get_and_install_input_package() {
+    if ! [[ $# -eq 1 ]]; then
+        log_fatal "deb_get_and_install_input_package() requires 1 argument"
+    fi
+    local package_path="$1"
+    local packages=$(ls ${package_path}/*.deb)
+
+    for package in $packages; do
+        log_info "Installing $package"
+        deb_install_package "${package}" "work/rootfs/"
```

Comment:
Installing the packages one by one has the limitation that if if package A depends on package B, it will break if we try to install package B first. We can solve this by passing all packages at ones to `dpkg --install`.

However how `deb_install_package` and related functions are designed they only accept one package at a time, so I won't make this a hard requirement, only something for you to consider. The task at hand is about mender-monitor and that will work fine.

:thinking: A simpler way to solve it would be for the user to pass a list of files rather than a directory, and then we assume that the user pass them in the correct order.

Reply:
Let me know what you think about this solution; i.e. instead of changing the functions to take many packages at once (which, as you said, is unrelated to this task) I added the possibility of manually setting the order if needed. So the script checks for the existence of a `package_order` file in `input package path` and installs the files listed first in the selected order, then continues to install the rest. Otherwise it just continues as before.

Reply:
It is a good solution! Nice idea.

Two comments:

I would name the file only `order`. Alternatively I would use `series` which is a common convention for applying source code patches in a directory, for example.

Also, it should be documented in the `mender_convert_config` file. Your commit message is good, you can copy paste from there.